plugins/env/app/views/samson_env/_environment_variables.html.erb

```diff
@@ -24,3 +24,16 @@
     <% end %>
   </div>
 <% end %>
+<%= link_to "Add row", "#", id: "add_environment_variable_row" %>
+
+<script>
+  $(function(){
+    $("#add_environment_variable_row").click(function(){
+      var $row = $(this).prev();
+      var $new_row = $row.clone();
+      $new_row.find('input').val('');
+      $row.after($new_row)
```

Comment:
missing semicolon is a little distracting.
could also pass the event and do `e.preventDefault();` instead of `return false;`